.mockery.yaml

```diff
@@ -0,0 +1,36 @@
+with-expecter: false
+disable-version-string: true
+outpkg: "mocks"
+dir: "{{.InterfaceDir}}/mocks/"
+filename: "{{ .InterfaceName | snakecase }}.go"
```

Comment:
do we need snakecase? I see most of our mocks just follow the interface name. If we have a single package with snakecase, we can just update it.